spring-cloud-azure-autoconfigure/src/test/java/com/microsoft/azure/spring/cloud/autoconfigure/context/AzureContextAutoConfigurationTest.java

```diff
@@ -34,15 +38,34 @@ public void testAzurePropertiesConfigured() {
                     assertThat(context.getBean(AzureProperties.class).getCredentialFilePath()).isEqualTo("credential");
                     assertThat(context.getBean(AzureProperties.class).getResourceGroup()).isEqualTo("group1");
                     assertThat(context.getBean(AzureProperties.class).getRegion()).isEqualTo("westUS");
+                    assertThat(context.getBean(TelemetryTracker.class)).isNotNull();
```

Comment:
This method is testing properties configured correctly. This assert seems not part of property.

Reply:
OK. I create new test for this.